ap-varawiki-content (#3)
gear-tech/vara-wiki: docs/intro.md

---
title: Vara Network
sidebar_position: 1
sidebar_label: Getting Started
---

## About Vara Network

The **Vara Network** is a stand-alone layer-1 decentralized network built and running on top of **[Gear Protocol](https://www.gear-tech.io/)**. 

Gear Protocol is a Substrate-based smart-contract platform that enables anyone to develop and run a dApp in Vara Network as well as other networks powered by the Gear Protocol’s runtime and technology.

> Vara Network is fast and scalable non-fork upgradable network that allows it to be a best playground for next-gen Gaming, Financial-based applications, experimental features but not only. Any other modern use cases are perfectly suit for running on Vara. Building on Vara Network is ideal for both developers already in Web3 as well as those migrating from Web2 seeking the most secure, efficient, scalable environment for deploying their decentralized applications.

Reviewer: ```suggestion
Vara Network is a fast and scalable non-fork upgradable network that allows it to be the best playground for next-gen Gaming, Financial-based applications, experimental features but not only. Any other modern use cases are perfectly suited for running on Vara. Building on Vara Network is ideal for both developers already in Web3 as well as those migrating from Web2 seeking the most secure, efficient, scalable environment for deploying their decentralized applications.
```

Author: ok

Many next-generation apps, like gaming and payments, require low latency execution. The Vara standalone network will provide the fastest on-ramp for these applications and will unlock new potential applications.

> Right after its start, Vara transitions though BETA stage. It means that as soon as it is live, there is no guarantee it is free of bugs. But thanks to fork-less upgrade, it will live forever and will never lose any user data.

Reviewer: ```suggestion
Right after its start, Vara transitions through the BETA stage. It means that as soon as it is live, there is no guarantee it is free of bugs. But thanks to the fork-less upgrade, it will live forever and will never lose any user data.
```

Author: ok

- Stable and upgradable network thanks to Substrate
- Little to no governance

Vara benefits from Gear’s unique tech stack that includes the Actor Model, Persistent Memory and the WebAssembly virtual machine (Wasm).

### Actor Model

> Anyone can instantiate a program that becomes a part of the network. In the Actor Model for communications these programs as well as users are known as “actors”. Actors do not share their state and communicate via messages, this way adding an extra layer of security to the network. All communications are asynchronous, but logically guaranteed to be handled, which increases the achievable network  speed and allows building of more sophisticated dApps.

Reviewer: ```suggestion
Anyone can instantiate a program that becomes a part of the network. In the Actor Model for communications these programs as well as users are known as “actors”. Actors do not share their state and communicate via messages, this way adding an extra layer of security to the network. All communications are asynchronous, but logically guaranteed to be handled, which increases the achievable network speed and allows building of more sophisticated dApps.
```

Author: removed it
